test/E2ETest/Microsoft.Test.E2E.AspNet.OData/NavigationPropertyOnComplexType/PeopleRepository.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Test.E2E.AspNet.OData.NavigationPropertyOnComplexType
+{
+    public class PeopleRepository
```

Comment:
>PeopleRepository [](start = 17, length = 16)

maybe add test for EF & EFCore

Reply:
EF and EFCore do not support navigation properties on complex types. 

---
In reply to: [295043430](https://github.com/OData/WebApi/pull/1738#discussion_r295043430) [](ancestors = 295043430)